### ISSUE TITLE
Philips 65PUS8535/12

### ISSUE THREAD

**Issue by carrerasnet** [PredatH0r/ChanSort#195]
Hi, would it be possible to include this Android TV in the Editor?

https://drive.google.com/folderview?id=14vCEdYAdy8E8U2woRQ4WAk7jAStcNII2

**Comment by PredatH0r** [comment]
The Philips "ChannelMap_110" format is already supported by ChanSort.
Did you encounter any problems when you tried to open the list?

You can pretty much select any of the .bin files in the folder structure, ChanSort will then try and read all the other files it needs.

**Comment by carrerasnet** [comment]
If I could open and modify.  But when I installed it on the TV it gave errors, it constantly changed channels.

**Comment by carrerasnet** [comment]
I have got it. I have done a re-installation of channels on the TV and I have not created any favorites. Then I used a reference list from a 28-inch samsung and the Philips already works perfectly.
Thanks for the program.

**Comment by nourbakhsh** [comment]
Hi,
I ran to the same issue on a Philips 58PUS8505 and could not find any information about it. It looks like that cable provider can lock the Position of channels so there won’t get sorted by ChanSort.

If anyone else runs to this issue, I solved it this way:
1.	Go to channel search menu
2.	Choose your Country
3.	**Do not choose any Provider**. Use the general search option so the channel position doesn’t get locked
4.	Export and edit with ChanSort as usual
